packages/react-focus-management/src/FocusManagementProvider.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+import { useMergedRefs } from '@fluentui/react-hooks';
+import { makeMergeProps } from '@fluentui/react-utils';
+import { getCurrentAbilityHelpers, createAbilityHelpers, Types as AHTypes } from 'ability-helpers';
+import { internal__FocusManagementContext, FocusManagementContextValue } from './focusManagementContext';
+
+type Dir = 'ltr' | 'rtl';
```

Comment:
nit: do we really need this type? It's not even exported...

Reply:
AH is not currently RTL aware, and that support needs to be contributed, but there will definitely need to be some Dir prop so that arrow keys are appropriately flipped

Reply:
I am totally agree about `dir` prop, but do we need a separate type? :) I.e.:

```tsx
export interface FocusManagementProvideProps extends React.HTMLAttributes<HTMLElement> {
   dir?: 'ltr' | 'rtl';
```

Reply:
ah ok, sure it's definitely possible to inline it